### PR TITLE
Adding select capabilities in activities tabs

### PR DESCRIPTION
**Pull request by mmattel**
Adding the capability of marking singe, multiple or all lines (ctrl-a) and copy it to the clipboard.

Backport to 2.9

@michaelstingl fyi

### CODE REVIEW

File: modules/ROOT/pages/navigating.adoc [hunk 1]
```diff
@@ -124,10 +124,21 @@ Use *Share with ownCloud* to see who you have shared with, and to modify their p
...
+Sync Protocol::
+Displays local activities such as which local folders your files went into.
+Not Synced::
+Shows errors such as files not synced because of being excluded or any other failing status.
+
+Double clicking an activity entry pointing to an existing file in tabs *Server Activities* or *Sync Protocol* will open the folder containing the file and highlight it.
 
 image:navigating/client-8.png[image, width=60%,pdfwidth=60%]
 
+You can mark in any of the activity tabs a single line, multiple lines or all lines with kbd:[CTRL+a] to copy the selected lines to the clipboard with menu:mouse[right-click > Copy to clipboard].
```
Comment (phil-davis) [inline modules/ROOT/pages/navigating.adoc:140]:
```suggestion
In any of the activity tabs you can mark a single line, multiple lines or all lines with kbd:[CTRL+a] and copy the selected lines to the clipboard with menu:mouse[right-click > Copy to clipboard].
```

File: modules/ROOT/pages/navigating.adoc [hunk 2]
```diff
@@ -124,10 +124,21 @@ Use *Share with ownCloud* to see who you have shared with, and to modify their p
...
+The Activity window contains the log of your recent activities, organized over three tabs:
+
+Server Activities::
+Includes new shares and files downloaded and deleted.
+Sync Protocol::
+Displays local activities such as which local folders your files went into.
+Not Synced::
+Shows errors such as files not synced because of being excluded or any other failing status.
+
+Double clicking an activity entry pointing to an existing file in tabs *Server Activities* or *Sync Protocol* will open the folder containing the file and highlight it.
```
Comment (phil-davis) [inline modules/ROOT/pages/navigating.adoc:136]: I have 2.9.1 on Ubuntu and the double-click is not working for me. I can right-click and select "Show in file browser".
Does double-click work for other people?

Comment (mmattel) [reply modules/ROOT/pages/navigating.adoc]: Good point, I hav eadopted the text to reflect this info

Comment (phil-davis) [reply modules/ROOT/pages/navigating.adoc]: Is double-click supposed to work? Is it a bug or a non-existent feature?

**Review by phil-davis — APPROVED** [review]
Sadly there is a difference between Windows and Linux. I guess if that is found to be a bug and gets fixed, we can easily update the docs again.

**Pull request merged**